kato/kato-azure/kato-azure.gradle

```diff
@@ -0,0 +1,14 @@
+dependencies {
```

Comment:
CF doesn't even have a mort module. I started putting the relevant bits in oort.